drghs/endpoints/maintnerd.yaml

```diff
@@ -13,203 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-swagger: "2.0"
-info:
-  description: "Google Cloud Endpoints for DevRel GitHub Services."
-  title: "DevRel GitHub Services API (TYPE)"
-  version: "1.0.0"
-host: "drghs.endpoints.PROJECT_ID.cloud.goog"
-x-google-endpoints:
-- name: "drghs.endpoints.PROJECT_ID.cloud.goog"
-  target: "IP_ADDR"
-consumes:
-- "application/json"
-produces:
-- "application/json"
-schemes:
-# Uncomment the next line if you configure SSL for this API.
-#- "https"
-- "http"
-paths:
-  "/update":
```

Comment:
With this path gone, DevRel Services is unable to update the repo list, that call is currently failing on DEV

Reply:
Yep; this is known

Reply:
This should be resolved with the new gRPC method